### PR TITLE
Bugfix/polkadot wasm fix alt

### PR DESCRIPTION
**Pull request by gre**
bugfix iterations that closes #208 to load the polkadot crypto in a lazy way.

We still have to test it.

### ❓ Context

- **Impacted projects**: LLC, LLM, LLD
- **Linked resource(s)**: [LIVE-2479](https://ledgerhq.atlassian.net/browse/LIVE-2479)

### ✅ Checklist

- [ ] **Test coverage**: no way to automate transactions yet for LLM.
- [x] **Atomic delivery**
- [x] **No breaking changes**

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/polkadot/polkadot-crypto.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import "@polkadot/wasm-crypto/initOnlyAsm";
```
Comment (elbywan) [inline libs/ledger-live-common/src/families/polkadot/polkadot-crypto.ts:1]: I think this is mobile-only, for desktop we might want to take the wasm version instead (which is the default one).

Comment (gre) [reply libs/ledger-live-common/src/families/polkadot/polkadot-crypto.ts:1]: oh ok, so possibly we move the **import** back to the reactNative entry then? but still keep the other code?

Comment (elbywan) [reply libs/ledger-live-common/src/families/polkadot/polkadot-crypto.ts:1]: yep I think this would do the trick 👍

**Review by haammar-ledger — APPROVED** [review]
:+1:

**Pull request merged**